microsoft-edge/devtools-guide-chromium/javascript/ado-symbol-server.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Debug original code using the Azure Artifacts symbol server
+description: Learn how to publish source maps to the Azure Artifacts symbol server from Azure DevOps to debug original source code in DevTools.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 02/24/2022
+---
+
+# Debug original code using the Azure Artifacts symbol server
+
+In production, the JavaScript code of your website may be different from the original source code you work with during development. It may have been compiled, minified, and bundled during your website's build step. Source maps are used to map your compiled code to your original source code. Using source maps makes it possible to view and debug your original source code in DevTools instead of your compiled code.
```

Comment:
Could we pls try one more iteration on Sentence 1?  This is an important discovery of a potential powerful pattern to apply across all pages/articles.  Sentence 1 should encapsulate the enlightenment that's given by comprehending these paragraphs 1 & 2.  Like a `<summary>` sentence at top of a function API Ref topic, that gets bubbled up to a parent page/table and is then read in isolation, to "sell" or "market" this distinctive function and what it does for you.  _After_ that, give the verbose background lead-up, = the `<remarks>` tag for /// .NET documentation-comments.

Anti-pattern (throughout the docs): do not start with telling a long-winded story, w/ slow buildup, and then deliver, finally, the payload.  Hit the user with payload first, lead with the selling point at start of Sentence 1 - which is the special, bubble-up, atomic, self-contained, enlightening, marketing tagline.  When you think of Product X / Feature X / Technology X, think of XYZ relevant-to-you, relatable-to-you benefit.  So:

Sentence 1:

To see and debug your original code in DevTools rather than seeing and trying to debug a very modified, minified, transformed, unrecognizable, garbled version of your code that's returned by the web server, use source maps served by the Azure Artifacts symbol server.    = .NET /// <summary> comment block that gets bubbled up to a parent table/topic and viewed in isolation.

Sentence 2/Paragraph 2: Background description of the scenario.  = .NET /// <remarks> comment block.

Please see `<summary>` vs. `<remarks>` distinction at https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/xmldoc/recommended-tags#summary